src/ui/atoms/chip/index.tsx

```diff
@@ -0,0 +1,84 @@
+import * as React from 'react';
+import styled, { StyledComponent } from 'styled-components';
+
+/**
+ * --rounding 
+ * --font-size
+ * --padding
+
+ * --chip-background-color — color of the background
+ * --chip-color — color of the text
+ * --chip-focus — color of the focus
+ */
+
+interface ChipProps {
+  children?: never;
+  icon?: string;
+  onClick?: (event: React.SyntheticEvent) => void;
+  onClose?: (event: React.SyntheticEvent) => void;
```

Comment:
```suggestion
  onIconClick?: (event: React.SyntheticEvent) => void;
```
Иконкой же может быть не только кнопка закрытия Chip

Reply:
Обсудили с @n1zee , в Chip идет только иконка закрытия.

Reply:
Тогда нет никакого практического смысла пробрасывать пропс иконка.

Но я знаю кейс, когда может быть chip с другой иконкой, например +, это подсказка по контактам или тегам, где по клику на чип можно его открыть, а по плюсу добавить.

Я думаю гибче будет сделать как-то так:

```ts
interface ChipProps {
  icon?: React.ReactNode;
  onIconClick?: React.MouseEvent<HTMLButtonElement>;
}
```